### ISSUE TITLE
Suggested Delay between video extractions

### ISSUE THREAD

**Issue by lucho1** [ahmedayman4a/Linkedin-Learning-Courses-Downloader#112]
It's not actually a requested feature, but rather a question (maybe a suggestion could be to add certain usage info - like a documentation page - in the readme?). I couldn't find an Issues template for a "question/doubt" so I used the "feature request" one since it's not as important as a bug.

What's the suggested delay between video extractions to avoid issues like #37? Is it the default one (2s)? Asking just to be able to continuosly download some courses without being "banned".

**Comment by secretlycarl** [comment]
hi, thanks for posting this. I also had the same issue as #37. I think what's more important is the amount of courses downloaded at once. I got through about 7 before I got temp banned. when I can DL again, I'm going to add a delay of an hour just to spread them out and let it run

edit: i thought the delay was for courses, not videos

**Comment by lucho1** [comment]
Yeah, it's for videos, to avoid being detected as a bot making too many requests to the server in a little time

**Comment by tentpiglet** [comment]
I realize this is an older thread, but did the issue ever get resolved as to what a proper delay would be? I recently downloaded a couple of courses with this utility and found myself banned for a day. I'd like to, obviously, prevent this from happening again :)
